src/web/mjs/connectors/Sukima.mjs

```diff
@@ -0,0 +1,191 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Sukima extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'sukima';
+        super.label = 'Sukima';
+        this.tags = [ 'manga', 'japanese' ];
+        this.url = 'https://www.sukima.me';
+        this.language = 'japanese';
+        this.requestOptions.headers.set( 'x-referer', this.url );
+        this.lock = false;
+    }
+
+    async _getMangas() {
+        let request = new Request(new URL('/api/book/v1/free/', this.url), {
+            method: 'POST',
+            body: '{"store":false,"genre":"0"}',
+            headers: {
+                'Content-Type': 'application/json;charset=utf-8'
+            }
+        });
+        let categorys = await this.fetchJSON(request);
+
+        let uri;
+        let pages;
+        let mangas = [];
+        for (const category of categorys.rows) {
```

Comment:
Category is a variable, so i recommend using `let`... also don't know if `const` is just syntactic JS sugar since it is still re-assignable opposed to behave like a real constant

Reply:
Since `const` is block scoped I try to introduce it wherever a variable does not change anymore. With emphasize on 'try'.